godtools/App/Features/AppLanguage/Presentation/LanguageSettings/LanguageSettingsViewModel.swift

```diff
@@ -44,7 +44,7 @@ class LanguageSettingsViewModel: ObservableObject {
         $appLanguage.eraseToAnyPublisher()
             .flatMap({ (appLanguage: AppLanguageDomainModel) -> AnyPublisher<ViewLanguageSettingsDomainModel, Never> in
                 
-                return self.viewLanguageSettingsUseCase
+                return viewLanguageSettingsUseCase
```

Comment:
Good catch!  That's a tricky distinction of capturing self vs. referencing the useCase within the scope.